src/status_im/ui/screens/wallet/views.cljs

```diff
@@ -35,6 +34,17 @@
       (:code currency)]]
     [react/text {:style styles/total-value} (i18n/label :t/wallet-total-value)]]])
 
+(defn- backup-seed-phrase []
+  [react/view styles/section
+   [react/touchable-highlight {:on-press #(re-frame/dispatch [:navigate-to :wallet-onboarding-setup])}
```

Comment:
it should lead to profile, no?

Reply:
no

Reply:
it should lead to the screen with wallet seed phrase

Reply:
https://github.com/status-im/status-react/issues/3941#issuecomment-389893518

Reply:
it still should lead to profile

Reply:
will do, but it's super confusing to be honest that these sketches were near onboarding sketches and they are not related to onboarding. and there was nothing about it in the description.